### PR TITLE
[Master] Add docs for HYPR authenticator

### PR DESCRIPTION
**Pull request by divyaamunugama**
Related issue: https://github.com/wso2/product-is/issues/16086

### CODE REVIEW

File: en/docs/guides/passwordless/hypr.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Passwordless login with HYPR
+
+[HYPR](https://www.hypr.com/) is a passwordless authentication method that allows users to log in to applications using biometrics on their devices. As a FIDO-certified platform, HYPR eliminates risks prevalent in password-based authentication, such as phishing and password reuse.
+
+This guide explains how you can use HYPR to add passwordless login to applications registered in the Identity Server.
+
+## Prerequisites
+
+- You need to configure the HYPR environment and have access to the HYPR control center. Learn how to do it in the [HYPR documentation](https://docs.hypr.com/hyprcloud/docs/cc-std).
+- You need to [set up the sample]({{base_path}}/guides/adaptive-auth/adaptive-auth-overview/#set-up-the-sample) application.
```
Comment (AfraHussaindeen) [inline en/docs/guides/passwordless/hypr.md:10]: The use of a SAML application could lead to the perception that this connector is exclusively compatible with apps based on SAML protocols. Therefore, I suggest including a note to clarify that this connector is also compatible with applications based on OIDC protocols.

File: en/docs/guides/passwordless/hypr.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Let's configure WSO2 IS with HYPR as a federated authenticator.
+
+1. On the Management console, go to **Main** > **Identity** > **Identity Providers** > **Add**.
+2. Enter an **Identity Provider Name**, **Display Name**, and **Description**.
+4. Expand the **Federated Authenticators** > **HYPR Configuration** and fill in the following details:
+
+    | Field name    | Description   |
+    |---------------|---------------|
+    | Enable        | Select this to enable HYPR authenticator for the login flow of the application    |
```
Comment (AfraHussaindeen) [inline en/docs/guides/passwordless/hypr.md:57]:
```suggestion
    | Enable        | Select this to enable HYPR authenticator for the identity provider.    |
```

File: en/docs/guides/passwordless/hypr.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    | Default       | Select this if the HYPR authenticator should be the default authenticator for the application    |
```
Comment (AfraHussaindeen) [inline en/docs/guides/passwordless/hypr.md:58]:
```suggestion
    | Default       | Select this if the HYPR authenticator should be the default authenticator for the identity provider.    |
```

File: en/docs/guides/passwordless/hypr.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    | Base URL      | This refers to the base URL you received from HYPR upon creating a tenant for your organization.  |
+    | Relying Party App ID  | This refers to the App ID you received for the application you created in the HYPR Control Center.    |
+    | API Token     | This refers to a new API token that is specifically generated for the HYPR App via the Control Center.    |
+
+    ![HYPR configurations]({{base_path}}/assets/img/guides/hypr-configurations.png)
+
+6. Click **Update** to save the configurations.
+
+## Add HYPR authenticator to the app
```
Comment (AfraHussaindeen) [inline en/docs/guides/passwordless/hypr.md:67]:
```suggestion
## Enable HYPR login for the application
```

File: en/docs/guides/passwordless/hypr.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    rewrite.custom_webapps=["/api/hypr/"]
+    ```
+
+### Deploy the HYPR login page
+
+To deploy the HYPR login page from the extracted artifacts folder, copy the `hyprlogin.jsp` and paste it to `<IS_HOME>/repository/deployment/server/webapps/authenticationendpoint` folder.
+
+## Configure HYPR as a federated authenticator
+
+Let's configure WSO2 IS with HYPR as a federated authenticator.
```
Comment (AfraHussaindeen) [inline en/docs/guides/passwordless/hypr.md:49]:
```suggestion
To configure WSO2 IS with HYPR as a federated authenticator.
```

File: en/docs/guides/passwordless/hypr.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+2. Enter an **Identity Provider Name**, **Display Name**, and **Description**.
+4. Expand the **Federated Authenticators** > **HYPR Configuration** and fill in the following details:
+
+    | Field name    | Description   |
+    |---------------|---------------|
+    | Enable        | Select this to enable HYPR authenticator for the login flow of the application    |
+    | Default       | Select this if the HYPR authenticator should be the default authenticator for the application    |
+    | Base URL      | This refers to the base URL you received from HYPR upon creating a tenant for your organization.  |
+    | Relying Party App ID  | This refers to the App ID you received for the application you created in the HYPR Control Center.    |
+    | API Token     | This refers to a new API token that is specifically generated for the HYPR App via the Control Center.    |
```
Comment (AfraHussaindeen) [inline en/docs/guides/passwordless/hypr.md:61]:
```suggestion
    | API Token     | This refers to the application's API access token that is generated in the HYPR Control Center.    |
```

File: en/docs/guides/passwordless/hypr.md (hunk 1 above)
Comment (divyaamunugama) [reply en/docs/guides/passwordless/hypr.md:10]: Shall we add it like this :arrow_down: 

`You need to set up your application. For this tutorial, we will be using the [sample SAML application]({{base_path}}/guides/adaptive-auth/adaptive-auth-overview/#set-up-the-sample) application. Note that you can use any application you wish to use.`

**Review by GANGANI — APPROVED** [review]

**Pull request merged**